### DISCUSSION TITLE
PEFT required??

### DISCUSSION THREAD

**Discussion by pstadnyk** [TencentARC/PhotoMaker#107] [Q&A]
I've installed the ComfyUI version of PhotoMaker but have run into the following error when queuing a prompt through the LoRALoader: File "/workspace/ComfyUI/venv/lib/python3.10/site-packages/diffusers/loaders/unet.py", line 592, in set_adapters raise ValueError("PEFT backend is required for 'set_adapters()'.")

Anyone else have this problem or a fix?

**Comment by Paper99** [comment]
pip install peft
